ForumProject/startups/tests.py

```diff
@@ -1,3 +1,160 @@
+from rest_framework_simplejwt.tokens import RefreshToken
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
 from django.test import TestCase
+from startups.models import Startup
+from users.models import CustomUser
+import random, string
 
-# Create your tests here.
+class StartupCreationTestCase(TestCase):
+    
+    def setUp(self):
+        '''
+        Setting up pre-requisites for testing 
+        '''
+        # Create a CustomUser for authentication
+        self.user = CustomUser.objects.create_user(
+            email='dummy@user.com',
+            first_name='John',
+            last_name='Doe',
+            phone_number='+3801234567',
+            password='password',
+            is_active=True
+        )
```

Comment:
You might consider using `setUpTestData` for creating data that does not change across tests, like the user object. This method runs once per class, not before each test method, potentially speeding up your tests.

Reply:
Done. Thank you - tests run time reduced by half!